Source/WebCore/rendering/RenderTheme.cpp

```diff
@@ -931,14 +932,14 @@ bool RenderTheme::isEnabled(const RenderObject& renderer) const
 
 bool RenderTheme::isFocused(const RenderObject& renderer) const
 {
-    Node* node = renderer.node();
-    if (!is<Element>(node))
+    auto element = dynamicDowncast<Element>(renderer.node());
+    if (!element)
         return false;
 
-    auto focusDelegate = downcast<Element>(*node).focusDelegate();
-    Document& document = focusDelegate->document();
+    RefPtr delegate = is<SliderThumbElement>(element) ? downcast<SliderThumbElement>(element) : element;
```

Comment:
This should be the `hostInput()` when the element is a slider thumb.

Reply:
Fixed, thanks